components/match_ticker/commons/match_ticker_display_components_new.lua

```diff
@@ -0,0 +1,366 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:MatchTicker/DisplayComponents/New
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+-- Holds DisplayComponents for the MatchTicker module
+-- It contains the new html structure intented to be use for the new Dota2 Main Page (for now)
+-- Will most likely be expanded to other games in the future and other pages
+
+local Abbreviation = require('Module:Abbreviation')
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Countdown = require('Module:Countdown')
+local DateExt = require('Module:Date/Ext')
+local LeagueIcon = require('Module:LeagueIcon')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Timezone = require('Module:Timezone')
+local StreamLinks = require('Module:Links/Stream')
+
+local HighlightConditions = Lua.import('Module:HighlightConditions')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+local OpponentDisplay = OpponentLibraries.OpponentDisplay
+
+local VS = 'VS'
+local SCORE_STATUS = 'S'
+local CURRENT_PAGE = mw.title.getCurrentTitle().text
+local HIGHLIGHT_CLASS = 'tournament-highlighted-bg'
+local TOURNAMENT_DEFAULT_ICON = 'Generic_Tournament_icon.png'
+local NOW = os.date('%Y-%m-%d %H:%M', os.time(os.date('!*t') --[[@as osdateparam]]))
+
+---Display class for the header of a match ticker
+---@class NewMatchTickerHeader
+---@operator call(string|number|nil): NewMatchTickerHeader
+---@field root Html
+local Header = Class.new(
+	function(self, text)
+		self.root = mw.html.create('div')
+			:addClass('infobox-header')
+			:wikitext(text)
+	end
+)
+
+---@param class string?
+---@return NewMatchTickerHeader
+function Header:addClass(class)
+	self.root:addClass(class)
+	return self
+end
+
+---@return Html
+function Header:create()
+	return mw.html.create('div'):node(self.root)
+end
+
+---Display class for matches shown within a match ticker
+---@class NewMatchTickerVersus
+---@operator call(table): NewMatchTickerVersus
+---@field root Html
+---@field match table
+local Versus = Class.new(
+	function(self, match)
+		self.root = mw.html.create('div'):addClass('versus')
+		self.match = match
+	end
+)
+
+---@return Html
+function Versus:create()
+	local bestof = self:bestof()
+	local scores = self:scores()
+	local upperText, lowerText
+
+	if bestof then
+		upperText = scores or VS
+		lowerText = bestof
+	elseif scores then
+		upperText = scores
+		lowerText = VS
+	end
+
+	if not lowerText then
+		return self.root:wikitext(VS)
+	end
+
+	return self.root
+		:node(mw.html.create('div')
+			:addClass('versus-upper')
+			:node(upperText or VS)
+		):node(mw.html.create('div')
+			:addClass('versus-lower')
+			:wikitext('(' .. lowerText .. ')')
+		)
+end
```

Comment:
could potentially integrate #4371 here too (linking to (big) match page from score if the match has such a page)

Reply:
I think it would be nice yes, will add it once the other PR is merged

Reply:
Should be ok now that we are reusing the existing Versus